di-ipv-credential-issuer-stub/src/test/java/uk/gov/di/ipv/stub/cred/handlers/AuthorizeHandlerTest.java

```diff
@@ -413,15 +422,26 @@ void generateResponseShouldPersistSharedAttributesCombinedWithJsonInput_withMiti
         when(httpResponse.statusCode()).thenReturn(200);
         when(httpClient.send(any(HttpRequest.class), any(HttpResponse.BodyHandler.class)))
                 .thenReturn(httpResponse);
+        when(mockSignedJwt.serialize()).thenReturn(DCMAW_VC);
+        when(mockVcGenerator.generate(any())).thenReturn(mockSignedJwt);
 
         authorizeHandler.generateResponse.handle(mockRequest, mockResponse);
 
-        verify(httpClient, times(1)).send(any(), any());
+        ArgumentCaptor<HttpRequest> requestArgumentCaptor =
+                ArgumentCaptor.forClass(HttpRequest.class);
+        verify(httpClient, times(1)).send(requestArgumentCaptor.capture(), any());
+        // This is a poor proxy for checking the content of the request. We can't access the content
+        // directly.
+        // The content is:
+        // '{"mitigations":["M01"],"vcJti","urn:uuid:e937e812-dafe-42e4-a094-6c9d41fee50c\n"}'
+        // If this changes, this test will fail.
+        assertEquals(79, requestArgumentCaptor.getValue().bodyPublisher().get().contentLength());
```

Comment:
Presumably we _could_ consume the body publisher and get the content, it's just a bit of a pain?

This does feel like a useful thing to test, but certainly not worth spending ages on - it'll be implicitly tested by our CIMIT integration tests anyway.

Reply:
Yeah, that might work. I'll merge this then dig into it a little.